policy_packs/aws/stack/deploy_aws_iam_stack/main.tf

```
@@ -0,0 +1,5 @@
+resource "turbot_policy_pack" "main" {
+  title       = "Deploy AWS IAM Stack"
+  description = "Deploy & manage AWS IAM resources using OpenTofu with AWS > IAM > Stack [Native]"
+  akas        = ["deploy_aws_iam_stack"]
```

```suggestion
  akas        = ["aws_iam_deploy_aws_iam_stack"]
```
Our standard naming convention for Policy Packs is that they start with `check` (Check mode only), `enforce` (Check and Enforce mode), `deny` (Preventive controls) or `enable` (for Guardrails managed stacks, e.g. event handlers).
Not sure if we should use `deploy_aws_iam_stack`?

Options:
1. `enforce_configure_aws_iam_resources_via_stack`
2. `enable_aws_iam_stack`
3. ?

---

@e-gineer and I discussed this one as well and believe we should use `deploy` for stacks.